src/main/java/com/github/swrirobotics/bags/BagController.java

```diff
@@ -209,8 +213,12 @@ public BagList getBags(@RequestParam String text,
         for (Bag bag : results.getBags()) {
             // The big grid doesn't need this information, and serializing it for
             // every bag will slow things down by a lot.
-            bag.getMessageTypes().clear();
-            bag.getTopics().clear();
+            if (!fillTypes) {
```

Comment:
If `fillTypes` and `fillTopics` were not explicitly provided as parameters in the request, this will be null, and so this line will throw a `NullPointerException`.  (so will line 219 below this)